modules/gdscript/gdscript_parser.cpp

```diff
@@ -72,6 +72,53 @@ bool GDScriptParser::_end_statement() {
 	return false;
 }
 
+void GDScriptParser::_parse_log(GDScriptParser::BlockNode *p_block, bool p_static, GDScriptParser::LogLevel log_level)
+{
+#ifdef DEBUG_ENABLED
+		// Store the logging category and message.
+
+		tokenizer->advance();
+
+		if (tokenizer->get_token() != GDScriptTokenizer::TK_PARENTHESIS_OPEN) {
+			_set_error("Expected '(' after debug_log");
+			return;
+		}
+
+		tokenizer->advance();
+
+		Vector<Node *> args;
+		const bool result = _parse_arguments(p_block, args, p_static);
+		if (!result) {
+			return;
+		}
+
+		if (args.size() != 2) {
+			_set_error("debug_log expects two arguments: category and message");
```

Comment:
Remember to update the messages to remove references to `debug_log` and instead refer to the correct logging function :slightly_smiling_face: